### PR TITLE
Switch from `toml` to `tomli` for TOML v1 compat

### PR DESCRIPTION
**Pull request by hukkin**
Closes https://github.com/nbQA-dev/nbQA/issues/621

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -35,7 +35,7 @@ py_modules = nbqa
 install_requires =
     ipython>=7.8.0
     tokenize-rt>=3.2.0
-    toml
+    tomli<2.0.0
```
Comment (hukkin) [inline setup.cfg:38]: You may want a different version constraint, but this is what I'd personally choose. You'll find my thoughts about this here https://github.com/psf/black/pull/2301#discussion_r647748572

**Comment by MarcoGorelli** [comment]
awesome, thanks! there's a docs failure, don't know if it's related, I'll take a look later

**Comment by hukkin** [comment]
:+1: Seems unrelated as `master` branch was failing too. I fixed it by upper pinning Sphinx

**Comment by MarcoGorelli** [comment]
cool, thanks!

**Comment by MarcoGorelli** [comment]
@all-contributors add @hukkin  for maintenance

**Pull request merged**